IBM/carbon-addons-iot-react
# Add PageTitleBar, deprecate PageHero

**Summary**

- Add PageTitleBar
  - Ported this from `Hero` in `1.x` branch as it's quite similar. I refactored so it falls in line with the [newly established pattern from design](https://pages.github.ibm.com/Watson-IoT/IoT-Design-Site/components/ui-shell-header-name/usage)
- Deprecate PageHero in favor of PageTitleBar

**Change List (commits, features, bugs, etc)**

- add reusable deprecation notice helper story (for consistency
- add deprecation notice to PageHero (brought in `warning` and the `dev-expression` plugin for `__DEV__` support.
- add PageTitleBar, stories, associated styles (refactored off styled-components), tests
- added a FullWidthWrapper helper component that we can use to wrap components in stories to make them full width (instead of flex/centered)
  - modified the table and tilecatalog stories to take advantage of this

**Related Issues**



- Closes #507 

**Acceptance Test (how to verify the PR)**

- View the PageHero story in storybook, if you pull down and inverse the __DEV__ var, it'll show the warning in the console.
- View the new PageTitleBar story, make sure it follows the [newly established pattern from design](https://pages.github.ibm.com/Watson-IoT/IoT-Design-Site/components/ui-shell-header-name/usage)
- View the table and tilecatalog stories - should be unaffected by the new fullwidth helper component
